docker/entrypoint.sh

```diff
@@ -12,4 +12,15 @@ if [ -n "$TAILSCALE_AUTHKEY" ]; then
   tailscale up --ssh --authkey=$TAILSCALE_AUTHKEY --hostname=$TAILSCALE_HOSTNAME
 fi
 source docker-entrypoint.sh
-_main $@
\ No newline at end of file
+
+PGUSER="${PGUSER:-$POSTGRES_USER}"
+echo "Now starting Omnigres database and application services. (${POSTGRES_INITDB_ARGS})"
+echo "You can connect to this service the following ways:"
+echo
+echo "    psql -h localhost -p ${PGPORT:-5432} -U ${PGUSER:-postgres} omnigres"
+echo "    http://localhost:8081/"
+echo
+echo "If you're using docker, verify the above ports with 'docker ps'"
```

Comment:
I like this idea! I wonder if we can retrieve the port mappings from within the container – or not worth it?

Arguably, this always applies:
> echo "If you're using docker, verify the above ports with 'docker ps'"

Because it's part of the docker image entrypoint.